src/main/java/org/mule/tools/maven/dependency/model/ClassLoaderModel.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.tools.maven.dependency.model;
+
+import org.mule.tools.maven.dependency.util.DependencyUtils;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static org.mule.tools.maven.dependency.util.DependencyUtils.isValidMulePlugin;
+
+public class ClassLoaderModel {
+
+  private String version;
+  private ArtifactCoordinates artifactCoordinates;
+  private SortedSet<Dependency> dependencies;
```

Comment:
Just to leave the dependencies in alphabetical order when we serialize, I will change the field declaration to just implement set and let the sortedset internal to the class